plans/k8s/eks-worker.yaml

```diff
@@ -0,0 +1,130 @@
+---
+    aws-k8s-worker:
+      required_secrets:
+        - path: lacework.access_token
+          description: lacework agent token
+        - path: lacework.api_url
+          description: lacework API url
+          default: https://api.lacework.net
+        - path: lacework.k8s_use_kubectl_agent
+          description: use legacy kubectl agent install instead of helm
+          default: false
+        - path: eks.enable_cluster_agent
+          description: enable collection agent
+          default: true
+        - path: lacework.deploy_agent
+          description: deploy lacework agent
+          default: true
+      steps:
+        - name: k8s
+          extension: EKS
+          description: This step uses the EKS extension (Terraform under the hood) to deploy the actual cluster. Expect runtimes in excess of 10 minutes
+          args:
+            region: !secret aws.region
+            cluster_name: workereks
+          helpers:
+            - helper: EKSKubectl
+              lifecycle: [ afterCreate ]
+
+        - name: lacework-agent-legacy
```

Comment:
probably no need for this entire block; or the conditions here or in the lacework-agent step

this was only left in the original k8s plan for compatibility